js/interface.js

```diff
@@ -2289,22 +2287,26 @@ $('#show-access-rules').click(function() {
   });
 });
 
+/**
+ * Check if security rule is present in main security rule list
```

Comment:
I don't understand what this function does when I read its description.
- The wording implies there will be an input (a security rule) but that does not seem to be the case
- What is the "main security rule list"?

Reply:
updated the description as per your suggestion and as per my understanding main security list is nothing but the current ds rules